### DISCUSSION TITLE
Can't index and see my photos

### DISCUSSION THREAD

**Discussion by chest069** [photoprism/photoprism#2071] [Q&A] [answered]
Hello,

I'm new to PhotoPrisim and seem to have run into an issue with it scanning/indexing my photos and could use some help.

I have installed PhotoPrisim using a Raspberry Pi and CasaOS for the docker implementation.

It shows the following : 

Volumes : 

/DATA/AppData/c3a9.....
/DATA/Gallery
/DATA/Media/pictures (this is where my windows pictures are mounted on my Pi) /pictures (this is the only Volume I created once it was installed

So when I say for PhotoPrisim to index I don't get anything imported or it show that they were indexed for me to see.

Do I have this setup wrong if so can someone help me correct it so it works please?

Thanks.

**Comment by lastzero** [comment, marked as answer]
We have just received our CasaOS test device, but won't have time for detailed tests before next week. Could be a memory issue, make sure you have enough swap and memory. See system requirements in the docs. Also check filesystem permissions:
https://docs.photoprism.app/getting-started/troubleshooting/
